docs/docs/policies.mdx

```diff
@@ -396,6 +398,308 @@ for details.
+To reduce false warnings, `IntentTEDPolicy` has two mechanisms in place at inference time:
+
+1. `IntentTEDPolicy`'s [priority](./policies.mdx#policy-priority) is intentionally kept lower than all
+[rule based policies](./policies.mdx#rule-based-policies). This ensures that if an `action_unlikely_intent`
+is predicted for the last intent, but there is a rule that can be triggered to predict
+the next action, the ensemble of policies is not affected by it and the winning prediction is
+still the one made by `RulePolicy`.
+
+2. `IntentTEDPolicy` does not predict an `action_unlikely_intent` if the last predicted intent
+isn't present in any of the training stories, which might happen if an intent is only used in rules.
+
+As mentioned earlier, while `TEDPolicy` learns the next best action, `IntentTEDPolicy` learns the
+next likely intent that a user can express. Given a conversation context, there can only be a single correct action,
+whereas there can be multiple intents that are likely to occur in the same conversation context. Hence,
+`IntentTEDPolicy` uses a multi-label cross entropy loss function during training. Moreover, it uses
+the [`IntentMaxHistoryTrackerFeaturizer`](#3-intent-max-history) to treat intents as target labels instead of assistant actions or utterances.
+
+#### Prediction of `action_unlikely_intent`
+
+`IntentTEDPolicy` is invoked immediately after a user utterance and can either
+trigger `action_unlikely_intent` or abstain (in which case other policies will predict actions).
+To determine if `action_unlikely_intent` should be triggered, `IntentTEDPolicy` computes a score
+ for the user's intent in the current dialogue context and checks if this score is below a
+ certain threshold score.
+
+This threshold score is learned at training time but can be adjusted at test time
+via the `tolerance` parameter.  The higher the `tolerance`, the lower the intent's
+score (the more unlikely the intent) needs to be before `IntentTEDPolicy` triggers
+the `action_unlikely_intent` action.
+
+**Configuration:**
+
+You can pass configuration parameters to the `IntentTEDPolicy` using the `config.yml` file.
+If you want to fine-tune model's performance, start by modifying the following parameters:
+
+* `epochs`:
+  This parameter sets the number of times the algorithm will see the training data (default: `1`).
+  One `epoch` is equals to one forward pass and one backward pass of all the training examples.
+  Sometimes the model needs more epochs to learn properly.
+  Sometimes more epochs don't influence the performance.
+  The lower the number of epochs the faster the model is trained.
+  Here is how the config would look like:
+
+  ```yaml-rasa title="config.yml"
+  policies:
+  - name: IntentTEDPolicy
+    epochs: 200
+  ```
+
+* `max_history`:
+  This parameter controls how much dialogue history the model looks at before making an inference.
+  Default `max_history` for this policy is `None`, which means that the complete dialogue history
```

Comment:
Why do we have `None` as the default?

I expect that the first thing we will do is recommend a more sensible value, like `5`. Why not make that the default?

Reply:
Just like `TED` we don't know what a sensible default is. Using the whole dialogue history does not make an assumption that only a limited amount of dialogue history is needed to make a prediction.

Reply:
Don't we know from experience that 5 is a reasonable default to start with, and then increase it when necessary?

I think we know that using the whole dialogue history is mostly not good.

Reply:
I agree with you, but we cannot change the defaults in a minor version. The defaults of `TEDPolicy` and `IntentTEDPolicy` will have to be changed together to keep them at a finite (and equal) value.